### ISSUE TITLE
Missing binary packages for 0.6.4

### ISSUE THREAD

**Issue by dolmen** [sgimenez/laby#33]
On https://sgimenez.github.io/laby/ the last Win32 package is for 0.6.2 while I see a 0.6.4 tag in this repo...

**Comment by sgimenez** [comment]
Yes, windows packages are not easy to build. The latest released version is lagging behind a bit.
